flytekit/clis/sdk_in_container/register.py

```diff
@@ -47,7 +47,7 @@ def register_tasks_only(project, domain, pkgs, test, version):
 
 @click.group('register')
 # --pkgs on the register group is DEPRECATED, use same arg on pyflyte.main instead
-@click.option('--pkgs', multiple=True, hidden=True)
```

Comment:
this is a temporary change that will cause a merge conflict on merge to master, just take master when the time comes.